argocd/metadata.csv

```diff
@@ -0,0 +1,152 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+argocd.api_server.go.gc.duration.seconds,gauge,,second,,A summary of the pause duration of garbage collection cycles in the API Server,0,argocd,api_server go gc duration seconds,
+argocd.api_server.go.gc.duration.seconds.count,count,,second,,The summary count of garbage collection cycles in the Repo Server API Server,0,argocd,api_server go gc duration seconds count,
```

Comment:
```suggestion
argocd.api_server.go.gc.duration.seconds.count,count,,second,,The summary count of garbage collection cycles in the API Server,0,argocd,api_server go gc duration seconds count,
```

Reply:
Looks like `Repo Server` may be here by mistake, but let me know if this is intentional